packages/react/src/components/Tooltip/next/DefinitionTooltip.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Popover, PopoverContent } from '../../Popover';
+import { match, keys } from '../../../internal/keyboard';
+import { usePrefix } from '../../../internal/usePrefix';
+
+const DefinitionTooltip = ({ children, definition }) => {
```

Comment:
Make sure to update this to a function declaration per our style guide

Reply:
I think we'll also want to add support for `...rest` on the inner button `<button>`